test/support/coverage.rb

```diff
@@ -0,0 +1,43 @@
+module Coverage
```

Comment:
no, it was only question. For me class is more clear than `module_function`, that's why I asked.